cmd/exec.go

```diff
@@ -213,6 +213,7 @@ func executeCommandInLocations(execCmd []string, locs []commandExecLocation, par
 		go io.Copy(ptmx, os.Stdin)
 		if parallel {
 			wg.Add(1)
+			loc := loc
```

Comment:
What's the intention here?

Reply:
capture the variable because it's used in the go routine below

Reply:
Right, why not pass the variable to the goroutine as a parameter to make it explicit?

Reply:
done